StratisCore.UI/main.ts

```diff
@@ -2,19 +2,31 @@ import { app, BrowserWindow, ipcMain, Menu, nativeImage, Tray } from 'electron';
 import * as path from 'path';
 import * as url from 'url';
 import * as os from 'os';
-if (os.arch() == 'arm') {
+if (os.arch() === 'arm') {
   app.disableHardwareAcceleration();
 }
 
+// Set to true if you want to build Core for sidechains
+const buildForSidechain = false;
+
 let serve;
 let testnet;
 let sidechain;
 let nodaemon;
 const args = process.argv.slice(1);
-serve = args.some(val => val === "--serve" || val === "-serve");
-testnet = args.some(val => val === "--testnet" || val === "-testnet");
-sidechain = args.some(val => val === "--sidechain" || val === "-sidechain");
-nodaemon = args.some(val => val === "--nodaemon" || val === "-nodaemon");
+serve = args.some(val => val === '--serve' || val === '-serve');
+testnet = args.some(val => val === '--testnet' || val === '-testnet');
+sidechain = args.some(val => val === '--sidechain' || val === '-sidechain');
+nodaemon = args.some(val => val === '--nodaemon' || val === '-nodaemon');
+
+if (buildForSidechain) {
+  sidechain = true;
+}
+
+let applicationName = 'Stratis Core';
```

Comment:
we can reduce it to `const applicationName = sidechain ? 'Cirrus Core'  :  'Stratis Core';`